hermes/probe/probe.go

```diff
@@ -1,25 +1,25 @@
 // Copyright 2020 Google LLC
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
+// you may not use this file except in copliance with the License.
```

Comment:
Typo due to global s/mp/p/. Here and elsewhere.

Reply:
Thanks for pointing this out, I've changed it accordingly.